libs/src/services/identity/index.js

```diff
@@ -120,8 +120,14 @@ class IdentityService {
    * @param {string} signatureData.signature
    * @param {string} signatureData.timestamp
    */
-  async logTrackListen (trackId, userId, listenerAddress, signatureData) {
-    const data = { userId }
+  async logTrackListen (
```

Comment:
for @hareeshnagaraj - plumb this to the tracks API layer, reference https://github.com/AudiusProject/audius-client/blob/master/src/services/AudiusBackend.js#L785